utils/documentation_tests.txt

```diff
@@ -55,3 +55,4 @@ src/transformers/models/wav2vec2/modeling_wav2vec2.py
 src/transformers/models/wav2vec2/tokenization_wav2vec2.py
 src/transformers/models/wav2vec2_with_lm/processing_wav2vec2_with_lm.py
 src/transformers/models/wavlm/modeling_wavlm.py
+src/transformers/models/ctrl/modeling_ctrl.py	
```

Comment:
Is this an intended change?

Reply:
Yes - during the [final merge](https://github.com/huggingface/transformers/pull/15529/files) this line was removed from `utils/documentation_tests.txt`. 

I wasn't able to track which exact commit in the linked PR that caused this change but I guess it happened because this line was added originally in one of the multiple rebases on main needed to pass CI (the PR was originally started in `4.17.dev0` 😅 ), so it was tracked as an additional but unrelated to debertav2 changes.

Restoring it for parity!

Reply:
Indeed!